hummingbot/strategy/pure_market_making/pure_market_making_v2.pyx

```diff
@@ -160,6 +162,10 @@ cdef class PureMarketMakingStrategyV2(StrategyBase):
     def sizing_delegate(self) -> OrderSizingDelegate:
         return self._sizing_delegate
 
+    @property
+    def sb_order_tracker(self):
```

Comment:
the "sb" prefix is really just there to indicate it's declared at the base class and avoid naming conflicts. For the user accessible property name I'd rather just name it "order_tracker"